definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -136,24 +110,24 @@
   {
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
-    "ID": "PBAReferenceLabel",
-    "Label": "Your pay account reference: ${PBAReference}",
+    "ID": "LabelPBAReference",
+    "Label": "Your pay account reference: ${FeeAccountReference}",
```

Comment:
This field also already exists: LabelSolicitorReference
<img width="543" alt="Screenshot 2020-08-17 at 11 52 53" src="https://user-images.githubusercontent.com/16866068/90388767-4073fb80-e080-11ea-8629-e5e3a4d9392b.png">

Reply:
Just for solicitor actually